apps/aecore/src/aec_chain_state.erl

```diff
@@ -277,6 +277,8 @@ is_key_block(N) -> node_type(N) =:= key.
 
 is_micro_block(N) -> node_type(N) =:= micro.
 
+node_header(#node{header = H}) -> H.
+
```

Comment:
just fyi, there was `export_header` (?!)

Reply:
Right. I was looking for it, but couldn't find it. I might fix this together with other review comments.

Reply:
`export_` name is kind of out of convention

Reply:
It used to indicate more than just getting a field, but the name just stuck on. If I change this, I'll probably go for `node_header/1` all around instead.